README.md

```diff
@@ -5,3 +5,12 @@ The contents are hostel at http://cse.buet.ac.bd/bsadd
 Also a latest blogs can be found on https://medium.com/bsadd
 
 Probable proxy location is http://bsadd.org/
+
+
+steps for inserting new session gallery: 
+    - create new folder and name it as you wish 
+    - put all images under that folder
+    - if you have thumbnails for your pictures, put them inside img/timg folder with the same name as original image folder. 
+    - insert necesary metadata in _data/gallery.yml file
+
+        (please note that if you don't have any thumbnails folder, you have to specify that in .yml by assigning empty string to thumbFolder variable i.e. `thumbFolder: ""` )
```

Comment:
Can we skip this step ?

Reply:
will try to incorporate that in [this](https://github.com/bsadd/bsadd-home/issues/7) issue

Reply:
yes I'll get rid of this part